### DISCUSSION TITLE
We need a quick switch-on/off for CoPilot suggestions

### DISCUSSION THREAD

**Discussion by v-ko** [community/community#67815] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

One of the things that really messes with me is that CoPilot is always on, and pollutes my thought process when writing comments and sometimes when I write code I don't need assistance on. It would be great to have a quick-switch action/shortcut to cover those cases. Currently there's the Enable/Disable action, but it involves a modal dialog "Do you want to switch of globally or just for X language?" , so it's not really adequate for what I'm talking about.

**Comment by Akash1134** [comment]
Hi @v-ko ,

Just want to sure here, are you referring to the use command "`github.copilot.toggleCopilot`" in [VSCode](https://docs.github.com/en/copilot/configuring-github-copilot/configuring-github-copilot-in-your-environment?tool=vscode#keyboard-shortcuts-for-macos-1) or you're using a different IDE? 

Thanks!

**Comment by v-ko** [comment]
Hi, @Akash1134 ,
That's the command I'm referring to, yes.
